common/src/main/java/org/apache/nemo/common/ir/IRDAG.java

```diff
@@ -650,6 +876,112 @@ private IREdge edgeToMessageAggregator(final IRVertex trigger,
     return newEdge;
   }
 
+  // These similar four helper methods are for inserting TaskSizeSplitterVertex
```

Comment:
Wouldn't it make more sense to keep the methods underneath inside TaskSizeSplitterVertex, as these methods are related to TaskSizeSplitterVertex?

Reply:
Makes sense to me! I'll modify the code as you suggested. Thanks.

Reply:
Makes sense to me either. I refactored the code as you mentioned. Thanks!